### ISSUE TITLE
Expected a JSON object

### ISSUE THREAD

**Issue by dairecunningham** [microsoft/PowerBI-Tools-For-Capacities#38]
Hi there,

anyone else getting an error at JSON.parse(reportParameters).reportUrl in the HTML file?
If I look at the PBIReport.json file in VSCode is looks to be malformed:
<img width="313" alt="JsonError" src="https://github.com/microsoft/PowerBI-Tools-For-Capacities/assets/8200788/f9a3cc1e-90a9-4150-b285-26f69ee9729e">
This is the sample in the screenshot but my file has all values filled.

Thanks in advance!
Daire

**Comment by furmangg** [comment]
That file is actually a JavaScript file. The file extension is a misnomer. 

If you can’t figure out how to make your file parse then feel free to share here for sanity check.

**Comment by dairecunningham** [comment]
> That file is actually a JavaScript file. The file extension is a misnomer.
> 
> I’d you can’t figure out how to make your file parse then feel free to share here for sanity check.

Yes, I noticed this actually. It seems to be an issue to do with specifying the filter parameters. I'm seeing issues if they are strings, have spaces or I want to omit altogether. But I can work around these thanks
